src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int BOUND = 100;
+    private static final String WHITE = "WHITE";
+    private static final int RADIUS = 10;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure generateFigure() {
+        switch (FigureType.values()[random.nextInt(FigureType.values().length)].name()) {
```

Comment:
make switch not on the String (name of the enum) but on the enum value itself; in `case` statements compare not with strings, but with enum values accordingly